src/core/Test_compare.ml

```diff
@@ -35,6 +36,16 @@ let cmp2sql = function
 
 let pp_cmp = Fmt.of_to_string cmp2sql
 
+let status_to_string = function
+  | `Sat -> {|
```

Comment:
why the newlines? :)

Reply:
To be honest, because I copied from `cmp2sql` above (: